src/ui-test/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { BottomBarPanel, InputBox, WebDriver, Workbench } from "vscode-uitests-tooling";
+
+export const TEST_ARRAY_RUN = [
+    "Routes startup",
+    "Hello Camel from yaml"];
+
+export const CAMEL_RUN_DEBUG_ACTION_LABEL = 'Run Camel Application with JBang and Debug';
+export const CAMEL_RUN_ACTION_LABEL = 'Run Camel Application with JBang';
+export const CAMEL_ROUTE_YAML_WITH_SPACE = 'demo route.camel.yaml';
+
+/**
+ * Executes a command in the command prompt of the workbench.
+ * @param command The command to execute.
+ * @returns A Promise that resolves when the command is executed.
+ */
+export async function useCommandInPalette(command: string): Promise<void> {
```

Comment:
I would go with naming from utils and from usage.. `executeCommand` seems right, WDYT?